www/themes/upd/upd.theme

```diff
@@ -142,3 +142,26 @@ function upd_preprocess_image(&$variables) {
     unset($variables['attributes']['height']);
   }
 }
+
+/**
+ * Implements hook_preprocess_node()
+ */
+function upd_preprocess_node(&$variables) {
+  $node = $variables['node'];
+  // If we are looking at the full view of a news article node.
+  if ($node->getType() == 'art' && $variables['view_mode'] == 'full') {
```

Comment:
Lets get this in a separate function so that if we add further node preprocessing in the future it doesn't get too messy.